libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -187,6 +187,7 @@ dictionary NodeState {
     u64 max_single_payment_amount_msat;
     u64 max_chan_reserve_msats;
     sequence<string> connected_peers;
+    u64 max_receivable_single_payment_amount_msat;
```

Comment:
Now that we add this field the former one (inbound_liquidity_msats) changes its semantic. 
My concern is that developers will upgrade and their logic will break without them knowing about it.
Can we rename the `inbound_liquidity_msats` so developers will get a compilation error on upgrade which will force them to change the code?
We can also leave it with the old semantic (and mark it as deprecated) and add another field (total_inbound_liquidity_msat) with the unified liquidity.
Anything that will not let this change go under the radar.

Reply:
That's a very good point @roeierez. I'll rename it to `total_inbound_liquidity_msat`.